package/yast2-sudo.spec

```diff
@@ -12,36 +12,35 @@
 # license that conforms to the Open Source Definition (Version 1.9)
 # published by the Open Source Initiative.
 
-# Please submit bugfixes or comments via http://bugs.opensuse.org/
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
 
 Name:           yast2-sudo
-Version:        4.1.0
+Summary:        YaST2 - Sudo configuration
+Version:        4.2.0
 Release:        0
+Url:            https://github.com/yast/yast-sudo
+Group:          System/YaST
+License:        GPL-2.0-only
 
-BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 Source0:        %{name}-%{version}.tar.bz2
 
-url:            http://github.com/yast/yast-sudo
-Group:          System/YaST
-License:        GPL-2.0-only
-Requires:	yast2-users
+Requires:       yast2-users
 # Wizard::SetDesktopTitleAndIcon
-Requires: yast2 >= 2.21.22
-#ycp::PathComponents
-Conflicts: yast2-core < 2.13.29
-#Sudo icons
-Conflicts: yast2_theme < 2.13.9
-BuildRequires:	yast2 yast2-users
-BuildRequires:  yast2-devtools >= 3.0.6
-BuildRequires:  rubygem(yast-rake)
+Requires:       yast2 >= 2.21.22
+Requires:       yast2-ruby-bindings >= 1.0.0
 
-BuildArchitectures:	noarch
+BuildRequires:  yast2 yast2-users
+BuildRequires:  yast2-devtools >= 4.2.2
+BuildRequires:  rubygem(yast-rake)
 
-Requires:       yast2-ruby-bindings >= 1.0.0
+#ycp::PathComponents
+Conflicts:      yast2-core < 2.13.29
```

Comment:
I think we can remove these Conflicts, that seems to be from the SLES10 times (or even older).